campaign_templates/forest.json

```diff
@@ -0,0 +1,173 @@
+{
+	"id": "oceans",
```

Comment:
You probably already know but some of these files have these "oceans" values, please change accordingly!